### DISCUSSION TITLE
Interpret Result of chi_square

### DISCUSSION THREAD

**Discussion by Srj** [pgmpy/pgmpy#1520] [Q&A] [answered]
````python
pgmpy.estimators.CITests.chi_square(X, Y, Z, data, boolean=True, significance_level = 0.05)
````

Suppose the return value of this function is <b>True</b>. Does this mean that X and Y are conditionally independent given Zs? I am confused about the return value of this function.

**Comment by ankurankan** [comment, marked as answer]
@Srj Yes, if the returned value is True, it means that X is conditionally independent of Y given Zs. Essentially, the null hypothesis of the test is that X is independent of Y given Zs, so if we get a p-value > 0.05, it returns independence (i.e. True) else rejects the null implying dependence.

**Reply by Srj** [reply]
Thanks for your quick response
